### PR TITLE
Ethnicity Rules Prof-15

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**

### CODE REVIEW

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 1]
```diff
@@ -10,20 +10,23 @@ import {
...
 import { formatTable } from '../../../_utils/formatters';
+import MultiSelect from '../../../_components/MultiSelect';
 
 export default class RaceAndEthnicity extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       checked: false,
       value: '',
+      other: [],
```
Comment (RamuKammagani-osi) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: Unnecessary State other: [] , Can be removed

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 2]
```diff
@@ -10,20 +10,23 @@ import {
...
       hispanic: [],
       selected: [],
       unableToDetermine: false,
       addOtherEthnicity: false,
       unableToDetermineReason: '',
       childClient: props.childClient,
     };
+    this.handleOtherEthnicity = this.handleOtherEthnicity.bind();
```
Comment (RamuKammagani-osi) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: Don't need to bind this function as we following the fat arrow pattern

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.test.js [hunk 3]
```diff
@@ -34,10 +31,10 @@ describe('<RaceAndEthnicity />', () => {
 
   describe('#handleOtherEthnicity()  function', () => {
     it('should an event handler that sets state', () => {
-      const expectedvalue = { myKey: 'foo', other_ethnicity: [] };
+      const expectedvalue = { other_ethnicity: { 0: 'Asian', 1: 'American' } };
```
Comment (kelfink) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.test.js]: typically expectedValue with a capital V

File: app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js [hunk 4]
```diff
@@ -73,17 +78,15 @@ export default class RaceAndEthnicity extends React.Component {
       this.state.childClient.primary_race === 'Unable to determine*'
     ) {
       return '';
-    } else if (ethnicities < 1) {
+    } else if (ethnicities.length < 1) {
```
Comment (kelfink) [inline app/javascript/Clients/clientIdPage/ClientInformation/RaceAndEthnicity.js]: as in other review, unit test needed for this change.

**Pull request merged**